packages/amplify-dotnet-template-provider/resources/lambda/Trigger/trigger-dynamodb.js

```diff
@@ -0,0 +1,10 @@
+exports.handler = function(event, context) {
```

Comment:
Looks like this and several other of the template JS files were copied from the node template package and should be removed?